packages/sdk/src/HopBridge.ts

```diff
@@ -1139,6 +1139,68 @@ class HopBridge extends Base {
     return (Date.now() / 1000 + this.defaultDeadlineMinutes * 60) | 0
   }
 
+  /**
+   * @readonly
+   * @desc The time slot for the current time.
+   * @param {Object} chain - Chain model.
+   * @param {Number} time - Unix timestamp (in seconds) to get the time slot.
+   * @returns {Object} Time slot for the given time as BigNumber.
+   */
+  public async getTimeSlot(
+    chain: TChain,
+    time: number
+  ): Promise<BigNumber> {
+    chain = this.toChainModel(chain)
+    const bridge = await this.getBridgeContract(chain)
+
+    return bridge.getTimeSlot(time)
+  }
+
+  /**
+   * @readonly
+   * @desc The challenge period.
+   * @param {Object} chain - Chain model.
+   * @returns {Object} The challenge period for the bridge as BigNumber.
+   */
+  public async challengePeriod(chain: TChain): Promise<BigNumber> {
+    chain = this.toChainModel(chain)
+    const bridge = await this.getBridgeContract(chain)
+
+    return bridge.challengePeriod()
+  }
+
+  /**
+   * @readonly
+   * @desc The size of the time slots.
+   * @param {Object} chain - Chain model.
+   * @returns {Object} The size of the time slots for the bridge as BigNumber.
+   */
+  public async timeSlotSize(chain: TChain): Promise<BigNumber> {
+    chain = this.toChainModel(chain)
+    const bridge = await this.getBridgeContract(chain)
+
+    return bridge.TIME_SLOT_SIZE()
+  }
+
+  /**
+   * @readonly
+   * @desc The amount bonded for a time slot for a bonder.
+   * @param {Object} chain - Chain model.
+   * @param {Number} timeSlot - Time slot to get.
+   * @param {String} bonder - Address of the bonder to check.
+   * @returns {Object} Amount bonded for the bonder for the given time slot as BigNumber.
+   */
+  public async timeSlotToAmountBonded(
+    chain: TChain,
+    timeSlot: number,
+    bonder: string
+  ): Promise<BigNumber> {
+    chain = this.toChainModel(chain)
+    const bridge = await this.getBridgeContract(chain)
```

Comment:
Remove `chain` argument and use `Chain.Ethereum` or `getL1Bridge` if the chain is always ethereum